apps/easypid/src/app/(app)/federation.tsx

```diff
@@ -0,0 +1,17 @@
+import { FunkeFederationDetailScreen } from '@easypid/features/wallet/FunkeFederationDetailScreen'
+import { useLocalSearchParams } from 'expo-router'
+
+export default function Screen() {
+  const { entityId, trustedEntityIds, name, logo } = useLocalSearchParams()
```

Comment:
We can provide the types here

```suggestion
  const { entityId, trustedEntityIds, name, logo } = useLocalSearchParams<{ entityId: string, etc...  }>()
```